cpp/include/cugraph/graph_functions.hpp

```diff
@@ -1038,6 +1048,7 @@ remove_multi_edges(raft::handle_t const& handle,
                    rmm::device_uvector<vertex_t>&& edgelist_dsts,
                    std::optional<rmm::device_uvector<weight_t>>&& edgelist_weights,
                    std::optional<rmm::device_uvector<edge_t>>&& edgelist_edge_ids,
-                   std::optional<rmm::device_uvector<edge_type_t>>&& edgelist_edge_types);
+                   std::optional<rmm::device_uvector<edge_type_t>>&& edgelist_edge_types,
+                   bool keep_min_value_edge = false);
```

Comment:
Shouldn't this flag also be exposed to the CAPI? In fact the new `cugraph_graph_create_sg` takes `drop_multi_edges` however the users on the higher stack (C, PLC,  python) won't be able to control this parameter.

Reply:
Nice catch, yes, if the graph is symmetric, we need to set keep_min_value_edge to true to maintain symmetry. See https://github.com/rapidsai/cugraph/pull/4054/commits/a68868f102b8cd9c04ee3a7c4f01e8fc5c6f50cf for the update.